java/src/main/java/tech/sourced/enry/Guess.java

```diff
@@ -0,0 +1,23 @@
+package tech.sourced.enry;
+
+/**
+ * Guess denotes a language detection result of which enry can be
+ * completely sure or not.
+ */
+public class Guess {
+    /**
+     * Result is the resultant language of the detection.
+     */
+    public String result;
+
+    /**
+     * Sure indicates whether the enry was completely sure the language is
+     * the correct one or it might not be.
+     */
+    public boolean sure;
```

Comment:
Not sure if it's a good idea to use different names for the same concepts in both `Go` and `Java`.
In Go, we're calling these `language` and `safe`, while here there are `result` and `sure`. Could we unify the naming? (maybe `language` and `sure` are the best.